samplesheets/tests/test_views_ajax_taskflow.py

```diff
@@ -531,7 +531,7 @@ def test_get_empty_coll(self):
                 data={'path': self.assay_path},
             )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.data['irods_data']), 0)
+        self.assertEqual(len(response.data['irods_data']), 1)
```

Comment:
Have to notice, that I do not fully understand why this values changed. The logic behind is calling 
```
with irods_backend.get_session() as irods:
    ret_data = irods_backend.get_objects(irods, self.path)
```
What responses with the data_objects. But I didn't chage anything from `irods_backend` (

Reply:
The length should indeed be 0. Can you print out the return data and paste it here in the commands? I assume it's still showing that one file which should've been deleted.

I don't believe I've changed anything in irodsbackend related to this, at least on purpose. Also, all other tests work as intended. We have upgraded to a new python-irodsclient version since you started, perhaps something has changed. Maybe it has something to do with using `unlink()`..

Reply:
Here is an output:
Data:
 {'irods_data': [{'name': 'test2', 'type': 'obj', 'path': '/sodarZone/projects/c7/c7ddf20e-0e36-4a00-8f9c-b5afb869f349/sample_data/study_7a0621aa-f8a8-4c0a-99bc-0bb154dbf165/assay_2ffd215a-390c-4e35-9431-f3641d5cd421/test2', 'size': 0, 'modify_time': '2023-06-20 10:31', 'irods_request_status': None, 'irods_request_user': None}]}